src/components/onboarding/index.tsx

```diff
@@ -174,7 +175,7 @@ export const Onboarding: React.FC = () => {
       );
     }
     // skip when going thru claiming trial growth
-    else if (claimedGrowth) {
+    else if (account?.trialPeriodEndedAt) {
```

Comment:
account can be async, is that OK here?

Reply:
This should be safe because there's a check at the top to see if account is not null (which occurs when we're loading). If this is an anti-pattern or if there's a better way to do this then I can update it though